drivers/lsm303dlhc/include/lsm303dlhc_params.h

```diff
@@ -47,7 +47,7 @@ extern "C" {
 #define LSM303DLHC_PARAM_ACC_SCALE      (LSM303DLHC_ACC_SCALE_4G)
 #endif
 #ifndef LSM303DLHC_PARAM_MAG_ADDR
-#define LSM303DLHC_PARAM_MAG_ADDR       (0x1e)
+#define LSM303DLHC_PARAM_MAG_ADDR       (0x1E)
```

Comment:
just noticed: I think in the majority of the RIOT code base we use small letters for hexadecimal numbers, aren't we?!

But doing `git grep 0x`, it seems that both are used. This might also be something to put in the coding conventions I guess, but we would need to vote about something.

Anyway, no blocker for this PR. I think I will simply open an issue for voting on a style so we can converge in the future...

Reply:
done: #8151

Reply:
I would personally go for the capital letters but I agree this is not consistent in the code base.